### PR TITLE
rename bitwise enum variables, following the pattern XxxFlags

### PR DESCRIPTION
**Pull request by Jinming-Hu**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

File: sdk/storage/azure-storage-blobs/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,8 @@
 - `PublicAccessType::Private` was renamed to `PublicAccessType::None`.
 - `startsOn` parameter for `GetUserDelegationKey` was changed to optional.
 - Return types of `BlobClient::StartCopyFromUri` and `PageBlobClient::StartCopyIncremental` were changed to `StartCopyBlobResult`, supporting poll operations.
+- `ListBlobContainersIncludeItem` was renamed to `ListBlobContainersIncludeFlags`.
+- `ListBlobsIncludeItem` was renamed to `ListBlobsIncludeFlags`.
```
Comment (katmsft) [inline sdk/storage/azure-storage-blobs/CHANGELOG.md:21]: Reminder to myself: Check if DataLake/File also needs this change.

**Comment by katmsft** [comment]
Make sure this change is in Guidelines, or I don't think we should do it at all.

**Comment by Jinming-Hu** [comment]
> Make sure this change is in Guidelines, or I don't think we should do it at all.

Not in guidelines doesn't mean we cannot do it.

**Comment by katmsft** [comment]
> > Make sure this change is in Guidelines, or I don't think we should do it at all.
> 
> Not in guidelines doesn't mean we cannot do it.

Is it acceptable for you that DataLake/File to not have this naming convention? I guess not. But not having this in guidelines is not convincing enough for me to make this change.

**Comment by Jinming-Hu** [comment]
> > > Make sure this change is in Guidelines, or I don't think we should do it at all.
> > 
> > 
> > Not in guidelines doesn't mean we cannot do it.
> 
> Is it acceptable for you that DataLake/File to not have this naming convention? I guess not. But not having this in guidelines is not convincing enough for me to make this change.

Hey, you said it yourself https://github.com/Azure/azure-sdk-for-cpp/issues/1133

**Comment by katmsft** [comment]
> > > > Make sure this change is in Guidelines, or I don't think we should do it at all.
> > > 
> > > 
> > > Not in guidelines doesn't mean we cannot do it.
> > 
> > 
> > Is it acceptable for you that DataLake/File to not have this naming convention? I guess not. But not having this in guidelines is not convincing enough for me to make this change.
> 
> Hey, you said it yourself #1133
> 
> image

I have second thoughts on this and engaged Rick/Jeffrey for the decision. Edited the reply.

**Review by katmsft — APPROVED** [review]

**Pull request merged**